snf-image-helper/inject-files.py

```diff
@@ -39,44 +40,58 @@ def timestamp():
     return current_time
 
 
-def parse_arguments(input_args):
-    usage = "Usage: %prog [options] <target>"
-    parser = OptionParser(usage=usage)
-    parser.add_option("-i", "--input",
-                      action="store", type='string', dest="input_file",
-                      help="get input from FILE instead of stdin",
-                      metavar="FILE")
-    parser.add_option("-d", "--decode", action="store_true", dest="decode",
-                      default=False,
-                      help="decode files under target and create manifest")
+def parse_arguments():
```

Comment:
Yes, this was done intentionally. The usage indicated that the <target> argument was required. Now, what's required and what's not depends on the combination of input arguments.